src/script/lua_api/l_object.cpp

```diff
@@ -647,17 +560,15 @@ int ObjectRef::l_get_bone_position(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
 	ObjectRef *ref = checkobject(L, 1);
-	ServerActiveObject *co = getobject(ref);
-	if (co == NULL)
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == nullptr)
 		return 0;
-	// Do it
-	std::string bone;
-	if (!lua_isnil(L, 2))
-		bone = readParam<std::string>(L, 2);
+
+	std::string bone = readParam<std::string>(L, 2);
```

Comment:
`bone` is also no longer optional (same in the function above)

Reply:
I don't think it should be, the function is literally called `get_bone_position`. Or is there something I'm missing @Jordach ? (If it is, there should be a comment somewhere)

Reply:
`""` is a valid way to refer to the root bone and `get_bone_position()` was a shortcut to achieve that previously.
It should continue to work.